app/src/main/java/org/covidwatch/android/ui/MainActivity.kt

```diff
@@ -39,8 +44,6 @@ class MainActivity : AppCompatActivity() {
         addDummyCEN()
 
         setContactEventLogging(true)
```

Comment:
Let's move this to After the button being pressed too? That can be done in another PR if you'd like, but I don't think we should start logging pre-permissions (can we even?)

Reply:
Done.